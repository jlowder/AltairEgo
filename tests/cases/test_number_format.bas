10 PRINT 1;TAB(8);2;TAB(16);3
20 PRINT 10;TAB(8);20;TAB(16);30
