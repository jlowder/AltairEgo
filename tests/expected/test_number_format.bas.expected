Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
 1      2       3
 10     20      30
OK
